src/theme/components/icon.ts

```diff
@@ -1,18 +1,25 @@
 import { defineStyleConfig } from "@chakra-ui/react";
 import { defineStyle, StyleFunctionProps } from "@chakra-ui/system";
 
-const svgBase = {
+const svgBase: {
+  display: string;
+  fill: string;
+  width: string;
+  height: string;
+} = {
   display: "inline-block",
   fill: "currentColor",
   width: "auto",
   height: "100%",
 };
-const align = {
+
+const align: Record<string, { marginEnd?: string; marginStart?: string }> = {
```

Comment:
Why was `Record` added here and elsewhere?

Reply:
see [here](https://github.com/NYPL/nypl-design-system/pull/1694#issuecomment-2457362609)

Reply:
I'm surprised we don't see more of these build errors since this pattern is used elsewhere.